tools/zetasql_helper/zetasql_helper/local_service/BUILD

```diff
@@ -24,6 +25,19 @@ cc_grpc_library(
     deps = [":local_service_cc_proto"],
 )
 
+cc_library(
+    name = "local_service",
+    srcs = ["local_service.cc"],
+    hdrs = ["local_service.h"],
+    copts = ["-Wno-sign-compare"],
```

Comment:
curious what is this used for?

Reply:
I took this compiler options from the ZetaSQL BUILD file. I guess it asks the compiler not to care about the sign of data types.